internal/service/advancedclustertpf/resource.go

```diff
@@ -99,80 +96,230 @@ func (r *rs) Update(ctx context.Context, req resource.UpdateRequest, resp *resou
 	if diags.HasError() {
 		return
 	}
+	api := r.Client.AtlasV220240805.ClustersApi
 	patchReq := update.PatchPayloadTpf(ctx, diags, &state, &plan, NewAtlasReq)
+	projectID := plan.ProjectID.ValueString()
+	clusterName := plan.Name.ValueString()
+	var cluster *admin20240805.ClusterDescription20240805
+	var advConfig *admin20240805.ClusterDescriptionProcessArgs20240805
+	var legacyAdvConfig *admin20240530.ClusterDescriptionProcessArgs
+	var err error
 	if patchReq != nil {
-		err := StoreUpdatePayload(patchReq)
-		if err != nil {
-			diags.AddError("error storing update payload", fmt.Sprintf("error storing update payload: %s", err.Error()))
+		pauseAfter := false
+		if patchReq.Paused != nil && patchReq.GetPaused() {
+			// More changes than pause, need to pause after
```

Comment:
is first part of the comment correct in the 2 places?  // More changes than pause